_sass/_classes.scss

```diff
@@ -29,8 +29,12 @@
 
 .class-content-container {
     display: grid;
-    grid-template-columns: 30% 70%;
+    grid-template-columns: 230px auto 70%;      //230px is the width of the desktop sidebar +10 px for padding since the ai-content card has no horizontal padding
```

Comment:
Thanks for adding the comment! I'm a bit confused - `$sidebar-width` is `300px`, how does that play into this?

Reply:
`$sidebar-width` is actually unrelated to the ai/ml sidebar. I think Leo defined this for the sidebar that he added to the dev setup page

Reply:
Oh I see, that's very confusing haha. Maybe something worth addressing later.